tOpt/pytorch_computer.py

```diff
@@ -13,8 +13,9 @@
     AbstractNNPComputer, AbstractEnergyAndGradHelper
 from tOpt.coordinates_batch import SameSizeCoordsBatch, CoordinateModelInterface
 from cddlib.chem.mol import BaseMol
-from tOpt.unit_conversion import Units
+from tOpt.unit_conversion import *
 
+import torchani
```

Comment:
Can we implement the torchani specific classes in a different .py file so if users can use them independently.